js/data.js

```diff
@@ -936,6 +1112,17 @@ var data = {
         dozent: ["Dr. rer. nat. Harald Sack", "Jörg Waitelonis", "Nadine Steinmetz", "Magnus Knuth"],
         nameLV: "Semantic Multimedia"
     },
+    semanticMultimedia2: {
+        kurz: "Semantic Multimedia<br />(BPET)",
```

Comment:
I just did, and the data is correct. We'll just stick to what the document says, and if someone bother he/she will have to ask Frau Pamperin, and then change it here, if wanted.